### ISSUE TITLE
Scripts are not working

### ISSUE THREAD

**Issue by sanfm12** [martakostova/pomodoro#30]
Hi,

I'm trying to use the following applescript on start, but when I try to test run, an error appears.

Applescript:
activate application "Timelime"
tell application "System Events"
	key code 7 using {option down} -- option-x toggle popup
end tell

Error:
Timelime got an error: Application isn’t running.

I tested with many other apps without success. 
Is this a bug? Can you work on it or is it dead because of sandboxing?

Simpler applescripts like the one bellow work fine, though: 
Say "good morning!" using "Daniel"

Thank you
![captura de tela 2015-04-24 as 20 13 14](https://cloud.githubusercontent.com/assets/7762205/7330152/a8c4cd3e-eabf-11e4-888d-468a20f89781.png)

**Comment by sanfm12** [comment]
Here is a clunky workaround to use applescripts in Timer:

Among the few scripts that Timer can execute, there is the "activate application "*app_name*"".
1- Create an app using automator, containing the applescript you want.
2- Make Timer activate that app.

For example, I created an app called "TimerOnStart" with the Applescript: 
*activate application "Timelime"*
*tell application "System Events"*
*key code 7 using {option down} -- option-x toggle popup*
*end tell*

And in Timer Script, on Start, I simply entered:
*activate application "TimerStart"*

Timer scrip helper may prompt an error, but it should run finely.
